scripts/release-packages.bash

```diff
@@ -23,3 +23,4 @@ npm publish dist/packages/nearfi --tag "${TAG}" --otp "${OTP}"
 npm publish dist/packages/opto-wallet --tag "${TAG}" --otp "${OTP}"
 npm publish dist/packages/welldone-wallet --tag "${TAG}" --otp "${OTP}"
 npm publish dist/packages/neth --tag "${TAG}" --otp "${OTP}"
+npm publish dist/packages/account-export --tag "${TAG}" --otp "${OTP}"
```

Comment:
I would like to know if this will publish the `account-export` package properly.

Reply:
At the moment the release process is quite manual which means one of the team members will have to do it, once we bump the version and merge everything to `main` then locally we would do a clean build of all the packages and run this script.

For first-time scoped npm packages, we would need to add an extra flag `--access=public` before we publish it.

```bash
npm publish dist/packages/account-export --tag "${TAG}" --otp "${OTP}" --access=public
```

I will ping you on slack and explain everything.